Update how-to-create-azure-support-request.md (#6)
MicrosoftDocs/azure-management-docs: articles/azure-portal/supportability/how-to-create-azure-support-request.md

1. Complete the **Contact info** section so that we know how to reach you.

  > [!TIP]

>   > By enabling "Save contact changes for future support requests", your personal contact info is saved for future use. However, please note that "Additional email for notification" is not saved.

Reviewer: ```suggestion
  > By enabling **Save contact changes for future support requests**, your personal contact info is saved for future use. However, the **Additional email for notification** value is not saved.
```
